sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -23,6 +27,18 @@
  */
 @Immutable
 public final class DefaultAzureCredential extends ChainedTokenCredential {
+    private static final String DEFAULT_CACHE_FILE_NAME = "msal.cache";
+    private static final Path DEFAULT_CACHE_DIRECTORY = Platform.isWindows() ?
+            Paths.get(System.getProperty("user.home"), "AppData", "Local", ".IdentityService", "msal.cache") :
+            Paths.get(System.getProperty("user.home"),".IdentityService", "msal.cache");
+    private static final String DEFAULT_KEYCHAIN_SERVICE = "Microsoft.Developer.IdentityService";
```

Comment:
Keychain service & account names are consistent with Python ext at https://github.com/AzureAD/microsoft-authentication-extensions-for-python/blob/71b5301cb8c4768a9f439bde20750d448062944c/msal_extensions/token_cache.py#L141 and this test in .NET ext: https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/blob/85c1e8ec283b940b1cc34eae44ac37fa645738ff/tests/Microsoft.Identity.Client.Extensions.Msal.UnitTests/MsalCacheStorageTests.cs